sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/ShardingMetaDataLoader.java

```diff
@@ -103,17 +104,25 @@ private SchemaMetaData loadDefaultSchemaMetaData() throws SQLException {
         return actualDefaultDataSourceName.isPresent()
                 ? SchemaMetaDataLoader.load(dataSourceMap.get(actualDefaultDataSourceName.get()), maxConnectionsSizePerQuery) : new SchemaMetaData(Collections.emptyMap());
     }
-    
-    // TODO check all meta data in once
+
     private void checkUniformed(final String logicTableName, final Map<String, TableMetaData> actualTableMetaDataMap) {
         ShardingTableMetaDataDecorator decorator = new ShardingTableMetaDataDecorator();
         TableMetaData sample = decorator.decorate(actualTableMetaDataMap.values().iterator().next(), logicTableName, shardingRule);
+        List<String[]> errorMetaDataList = new LinkedList<>();
```

Comment:
The design `List<String[]>` is not clear, could you consider about define an object to process it?

Reply:
yeah, from understanding and readable, I think define an object is better.

Reply:
I've defined a new class name of "TableMetaDataViolation" for upon problem, review again, pls, thanks.